package.json

```diff
@@ -140,13 +140,20 @@
       {
         "command": "github-actions.workflow.logs",
         "category": "GitHub Actions",
-        "title": "View logs",
+        "title": "View job logs",
         "when": "viewItem =~ /job/",
         "icon": {
           "dark": "resources/icons/dark/logs.svg",
           "light": "resources/icons/light/logs.svg"
         }
       },
+      {
+        "command": "github-actions.step.logs",
+        "category": "GitHub Actions",
+        "title": "View step logs",
+        "when": "viewItem =~ /step/",
+        "icon": "$(globe)"
```

Comment:
We are copying the above section for View job logs and adding `"command": "github-actions.step.logs"` everywhere on the page that has "command": "github-actions.workflow.logs", since this new button for step logs needs to be anywhere workflow logs is.